saleor/graphql/menu/bulk_mutations.py

```diff
@@ -26,3 +27,9 @@ class Meta:
         description = "Deletes menu items."
         model = models.MenuItem
         permissions = ("menu.manage_menus",)
+
+    @classmethod
+    def bulk_action(cls, queryset):
+        menu_ids = set([item.menu_id for item in queryset])
```

Comment:
What about directly creating a set?

```suggestion
        menu_ids = {item.menu_id for item in queryset}
```